src/index.d.ts

```diff
@@ -0,0 +1,65 @@
+import * as resultTypes from "./types";
+
+declare namespace PlantUMLParser {
+  export class Class {
+    constructor(name: string, title: string, isAbstract: boolean, members: object[]);
```

Comment:
These properties are all non private. Therefore wen migrating this to `types.ts` I would suggest the following constructor (or similar):

```js
constructor(
  public name: string,
  public title: string,
  public isAbstract: boolean = false,
  public members: (MemberVariable | Method)[] = [],
){
  this.isAbstract = isAbstract || false;
  this.members = members || [];
}
```

Reply:
Noted, thanks